packages/graphql-hooks-memcache/index.test.js

```diff
@@ -0,0 +1,5 @@
+describe('ssr', () => {
+  it('does stuff', () => {
```

Comment:
Maybe add something a little worthwhile, even that a component render for example.

Reply:
This package currently has no tests, this was added to make sure the new test set up ran everything from root. I'll remove this & add raise some issues to add test coverage to the ssr & memcache packages once this is merged to avoid bloating this PR too much.